Gems/ROS2/Code/Source/RobotImporter/Pages/CheckAssetPage.cpp

```diff
@@ -139,4 +149,22 @@ namespace ROS2
         }
     }
 
+    void CheckAssetPage::RefreshTimerElapsed(){
+        for (int i =0; i < assetsId.size(); i++)
+        {
+            const auto &assetId  = assetsId[i];
+            if (processedAssets.contains(assetId))
+            {
+                continue;
+            }
+            const AZStd::string productRelPathVisual = Utils::GetModelProductAsset(assetId);
+            const AZStd::string productRelPathCollider = Utils::GetPhysXMeshProductAsset(assetId);
+            QString text = QString::fromUtf8(productRelPathVisual.data(), productRelPathVisual.size())+ " " +
+                QString::fromUtf8(productRelPathCollider.data(), productRelPathCollider.size());
+            bool isOk = productRelPathVisual.empty() || productRelPathCollider.empty();
+            m_table->setItem(i, 4, createCell(isOk, text));
```

Comment:
The timer will continue hitting over and over, is it okay for this code to run over and over?  It depends on whether processedAssets.contains(that assetid), but does not add it to the processedAssets array, so its going to be constantly setting the item?